tests/unittests/server/routes/test_camera.py

```
@@ -0,0 +1,765 @@
+        self,
+        batch_intrinsics_endpoint,
+        mock_batch_request,
+        multi_sample_dataset,
+        multi_sample_ids,
+        multi_dataset_id,
+    ):
+        """Tests that missing samples return error in results, not raise."""
+        from bson import ObjectId
+
+        bad_id = str(ObjectId())
+        sample_ids = multi_sample_ids + [bad_id]
```

Ruff (0.14.10)

Consider `[*multi_sample_ids, bad_id]` instead of concatenation

Replace with `[*multi_sample_ids, bad_id]`

(RUF005)